_includes/tutorials/produce-consume-lang/scala/markup/dev/3_4-schema-serde.adoc

```diff
@@ -0,0 +1,21 @@
+The case class instances in our code will be serialized in an https://avro.apache.org/[Avro] format.
```

Comment:
For your consideration: This is a great description of what's going on with the serdes. But there's a lot of information to absorb in the `Create a schema for the events` section. Maybe break the serde part into a separate section in the `scala.yml` file?